mlir/include/mlir/Dialect/MIOpen/Tuning/GridwiseGemmParams.h

```diff
@@ -312,9 +85,6 @@ struct PerformanceImplicitGemmV4R4Fwd
   std::tuple<std::size_t, LogicalResult>
   CalculateLdsNumberOfByte(const ConvolutionContext &ctx) const;
```

Comment:
I figured that majority of those utility functions are same across xdlops solvers, or across non-xdlops solvers. Is it possible to add a comment before each function like:

> This function has the same implementation acorss all non-xdlops solvers

Just so that we become aware of the the evolve-ment of different solvers.